themes/lcars.yaml

```diff
@@ -199,10 +199,17 @@
     ha-card.header-right {
       background: var(--lcars-card-top-color) !important;
       text-transform: uppercase;
-      border-radius: 40px 0px 0px 0px !important;
+      border-radius: 0px 40px 0px 0px !important;
       border-top: 10px solid var(--lcars-card-top-color);
       border-right: 35px solid var(--lcars-card-top-color);
     }
+    ha-card.header.right > ha-markdown.no-header {
```

Comment:
Should this be `ha-card.header-right` instead of `ha-card.header.right`?

Reply:
fixed.